lms/djangoapps/grades/tasks.py

```diff
@@ -61,13 +62,41 @@ class _BaseTask(PersistOnFailureTask, LoggedTask):  # pylint: disable=abstract-m
 
 
 @task(base=_BaseTask)
-def compute_grades_for_course(course_key, offset, batch_size):
+def compute_grades_for_course_with_kwargs(course_key, offset, batch_size, **kwargs):
```

Comment:
hmm.. I thought we had noted to use kwargs in the PR where this task was introduced - we may have missed it.
Can you put all the parameters inside the `kwargs`?  This is what we're doing with the grades tasks and have found it useful for forward compatibility.